### PR TITLE
PLT-6117 Fixed erroneous option for `marlowe-proxy` in operables.

### PR DESCRIPTION
**Pull request by bwbush**
Pre-submit checklist:
- Branch
    - [ ] Tests are provided (if possible)
    - [ ] [Test report is updated](https://github.com/input-output-hk/marlowe-cardano/blob/main/marlowe/test/test-report.md) (if relevant)
    - [x] Commit sequence broadly makes sense
    - [x] Key commits have useful messages
    - [x] Relevant tickets are mentioned in commit messages
    - [x] Formatting, PNG optimization, etc. are updated
- PR
    - [x] Self-reviewed the diff
    - [x] Useful pull request description
        - Review required
        - [ ] Substantial changes to code, test, or documentation
        - [ ] Change made to Marlowe validator (@bwbush and @palas must be included as reviewers)
        - Review not required
        - [x] Minor changes to non-critical code, documentation, nix derivations, configuration files, or scripts
        - [ ] Formatting, spelling, grammar, or reorganization
    - [ ] Reviewer requested

### CODE REVIEW

File: deploy/nomadTasks.nix [hunk 1]
```diff
@@ -169,6 +169,7 @@ rec
     env = {
       HOST = "0.0.0.0";
       PORT = "\${NOMAD_PORT_marlowe_load}";
+      QUERY_PORT = "\${NOMAD_PORT_contract_query}";
```
Comment (bwbush) [inline deploy/nomadTasks.nix:172]: @jhbertra , will this addition break the Nomad infrastructure or require an addition to it?

Comment (jhbertra) [reply deploy/nomadTasks.nix:177]: I think it should be fine as long as you add the `"contract_query"` to the `config.ports` below.

Comment (jhbertra) [reply deploy/nomadTasks.nix:177]: I think it should be fine as long as you add the `"contract_query"` to the `config.ports` below.

File: deploy/operables.nix [hunk 2]
```diff
@@ -170,7 +171,6 @@ in
 
       ${wait-for-socket}/bin/wait-for-socket "$CARDANO_NODE_SOCKET_PATH"
 
-      export OTEL_SERVICE_NAME=marlowe-chain-indexer
```
Comment (jhbertra) [inline deploy/operables.nix:173]: Let's keep the default values for these:

```suggestion
export OTEL_SERVICE_NAME="${OTEL_SERVICE_NAME:-marlowe-chain-indexer}"
```

**Review by jhbertra — APPROVED** [review]
Thanks!

File: deploy/operables.nix (hunk 2 above)
Comment (bwbush) [reply deploy/operables.nix:173]: @jhbertra , Would that force the need to use OTEL, or would `OTEL_SERVICE_NAME` be harmless?

Comment (bwbush) [reply deploy/operables.nix:173]: @jhbertra , Would that force the need to use OTEL, or would `OTEL_SERVICE_NAME` be harmless?

File: deploy/nomadTasks.nix (hunk 1 above)
Comment (bwbush) [reply deploy/nomadTasks.nix:172]: Fixed

**Pull request merged**